types/ember__string/index.d.ts

```diff
@@ -1,18 +1,18 @@
 // Type definitions for non-npm package @ember/string 3.0
 // Project: https://emberjs.com/api/ember/3.4/modules/@ember%2Fstring
 // Definitions by: Mike North <https://github.com/mike-north>
+//                 Scott Kidder <https://github.com/scottkidder>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import { SafeString } from "./-private/handlebars";
+/** @deprecated use imports from '@ember/template' instead. */
+export { htmlSafe, isHTMLSafe } from '@ember/template';
```

Comment:
I'm unclear here -- is this

(a) a bug fix in our types, where `import { htmlSafe } from '@ember/template';` no longer works entirely in Ember 3.x

or 

(b) A deprecation in Ember 3.x that currently works, but logs a deprecation warning?

Reply:
@mike-north as far as I can tell `import { htmlSafe } from '@ember/string';` still works and is not deprecated. However official [docs](http://api.emberjs.com/ember/3.13/functions/@ember%2Ftemplate/htmlSafe) say to import from `@ember/template`.

I imagine that the import from `@ember/string` should/will be deprecated at some point.

When I used `@deprecated` here I was thinking that I was deprecating the dts import itself. I'm happy to remove `@deprecated` to revisit at a later date since there is no underlying deprecation to match.

Reply:
Those were my findings as well. I guess remove `@deprecated`. Here's where they changed the docs: https://github.com/emberjs/ember.js/pull/16887 (not much info there)

Reply:
Here we go: https://github.com/emberjs/rfcs/blob/master/text/0236-deprecation-ember-string.md

Reply:
We need to align with Ember's public API. If it's not deprecated there, we should not indicate anything is deprecated here.

Reply:
Got it, makes sense. Thoughts on if it deserves another RFC to deprecate the exports from `@ember/string`?

Reply:
It's unclear whether
> 1. Move` htmlSafe` and `isHTMLSafe` to `@ember/template`

(from [the above RFC](https://github.com/emberjs/rfcs/blob/master/text/0236-deprecation-ember-string.md))
would include deprecating the exports from `@ember/string`. Regardless, the current state appears to be that importing from `@ember/string` not yet officially deprecated, just no longer being taught in the guides. Therefore, I agree with @mike-north that we shouldn't indicate deprecation here at this time.